samples/unit-tests/indicator-price-indicator/recalculations/demo.js

```diff
@@ -185,6 +185,10 @@ function (assert) {
         controller = new TestController(chart),
         button = chart.stockTools.listWrapper.childNodes[0].childNodes[0];
 
+    document.getElementsByClassName(
```

Comment:
In tests, it's better to use `chart.container.getElementsByClassName()` instead of `document.getElementsByClassName()`, because we have templates available in karma (which are separate charts). Alternatively, you can use `chart.stockTools.wrapper` which should be a reference directly to this DOM element.